<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1591</width>
    <height>912</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Video Cutter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffffff;&quot;&gt;Load Video&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>790</y>
      <width>1280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>50</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>850</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon_play.png</normaloff>icon_play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_frame_index">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>850</y>
      <width>144</width>
      <height>48</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>(0/0)</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scene_start">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>850</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>[</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scene_end">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>850</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>]</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scene_save">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>850</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon_save.png</normaloff>icon_save.png</iconset>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>10</y>
      <width>271</width>
      <height>251</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scene_init">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>850</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Init</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_file_open">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>850</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon_file_open.png</normaloff>icon_file_open.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_scene_progress_bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>1280</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scene_remove">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>850</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon_remove.png</normaloff>icon_remove.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxRhythm">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>720</y>
      <width>271</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Rhythm</string>
    </property>
    <widget class="QRadioButton" name="radioButtonRhythmSinus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sinus Rhythm</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRhythmArrh">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Arrhythmia</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonImgOther_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Others</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRhythm">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxImg">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>610</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Image Quality</string>
    </property>
    <widget class="QRadioButton" name="radioButtonImgGood">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>68</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Good</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonImgBad">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>68</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Bad</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonImgOther">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Others</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditImg">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxECG">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>280</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>ECG Quality</string>
    </property>
    <widget class="QComboBox" name="comboBoxECG">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>First part</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Second part</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Third part</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fourth part</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fifth part</string>
      </property>
     </item>
    </widget>
    <widget class="QRadioButton" name="radioButtonECGBad">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>68</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Bad</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonECGGood">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>68</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Good</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonECGOther">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Others</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditECG">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonECGOk">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetECG">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>141</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonECGRemove">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon_remove.png</normaloff>icon_remove.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
